scripts/genome_process.sh

```diff
@@ -57,37 +51,22 @@ rm -fr "$logFile"; touch "$logFile"
 ###  MAIN  ###
 ##############
 
-## GET & FILTER GENE ANNOTATIONS
-
-# Get genome sequences fasta files
-echo "Downloading genome sequences files..." >> "$logFile"
-# wget -i "${genomeSeqURLs}" --output-document "${rawDir}/${fileNamePrefix}.genome.fa.gz"
-# genomeSeq="${resDir}/${fileNamePrefix}.genome.fa.gz"
-
-for url in "${genomeSeqURLs[@]}"; do
-    wget "$url" --output-document "${rawDir}/$(basename "$url")" &> /dev/null
-done
-
-# Concatenate genome sequences fasta files
-echo "Concatenating genome sequences files..." >> "$logFile"
-genomeSeq="${resDir}/genome.fa.gz"
-for url in "${genomeSeqURLs[@]}"; do
-    cat "${rawDir}/$(basename "$url")" >> "$genomeSeq"
-done
-
-# Trim genome sequences IDs
+## TRIM GENOME SEQUENCES IDs
 echo "Triming genome sequenes IDs..." >> "$logFile"
```

Comment:
I know you didn't do this and it isn't strictly a concern of _this_ issue, but there are two typos here, and given that you change a lot right before and after this line, you could also fix there here, I think:
- `Trimming` instead of `Triming`
- `sequence` instead of `sequenes`

Reply:
Ignore this, see next comment.